src/app/language-support/page.tsx

```diff
@@ -37,25 +93,17 @@ export default function Page() {
             <th>id</th>
             <th>title</th>
             <th>summary</th>
+            <th>language</th>
             <th>is_remote</th>
+            <th>listing_type</th>
             <th>deadline</th>
-            <th>language</th>
+            <th>upcoming_date</th>
+            <th>num_months</th>
+            <th>hours_per_month</th>
             <th>num_pages</th>
           </tr>
         </thead>
-        <tbody>
-          {languageSupport.map(l => (
-            <tr key={l.id}>
-              <td>{l.id}</td>
-              <td>{l.title || 'null'}</td>
-              <td>{l.summary || 'null'}</td>
-              <td>{getIsRemoteValue(l.is_remote)}</td>
-              <td>{l.deadline}</td>
-              <td>{l.language}</td>
-              <td>{l.num_pages || 'null'}</td>
-            </tr>
-          ))}
-        </tbody>
+        <tbody>{allLanguageSupport.map(l => allLangSupportRow(l))}</tbody>
```

Comment:
Looks like we're getting a console error from not having a key for all the mapped items, could we add the id as a key?)
<img width="371" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/b73b63cb-20f2-44f2-b785-0bb795608e4b">